src/general/disallowanonymous/src_test.rego

```diff
@@ -222,6 +222,18 @@ test_allowed_multiple_role_multiple_subjects_mix_rolebinding {
     count(results) == 0
 }
 
+test_authenticated_with_disallow_authenticated {
+    inp := {"review": rolebinding([{"name": "system:authenticated", "kind": "Group"}, {"name": "system:anonymous", "kind": "User"}], "role-2"), "parameters": { "allowedRoles": [], "disallowAuthenticated": true }}
+    results := violation with input as inp
+    count(results) == 1
+}
+
+test_authenticated_with_disallow_authenticated_false {
+    inp := {"review": rolebinding([{"name": "system:authenticated", "kind": "Group"}, {"name": "system:anonymous", "kind": "User"}], "role-2"), "parameters": { "allowedRoles": [], "disallowAuthenticated": false }}
+    results := violation with input as inp
+    count(results) == 0
```

Comment:
This should still raise a violation due to the `system:anonymous` subject?

Reply:
The review object is the `authenticated_rolebinding`, which only has `system:authenticated` as a subject.